tests/test_inner.py

```diff
@@ -779,9 +828,13 @@ def test_workflow_docker_build_error():
                                                   }}],
                                    plugin_files=[this_file])
 
-    assert workflow.build_docker_image().is_failed()
-
+    try:
+        workflow.build_docker_image()
+    except Exception as ex:
+        pass
```

Comment:
yeah I need build plugins to raise exception, so I can check that only exit plugins are run after